blocks/inspector-controls/checkbox-control/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from 'components';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from './../base-control';
+import './style.scss';
+
+function CheckboxControl( { label, heading, checked, instanceId, onChange, ...props } ) {
+	const id = 'inspector-checkbox-control-' + instanceId;
+	const onChangeValue = ( event ) => onChange( event.target.value );
+
+	return (
+		<BaseControl label={ heading } id={ id }>
+			<label className="blocks-checkbox-control__label" htmlFor={ id }>
```

Comment:
I see! I'm wondering if it's ok for an input to have two labels? Maybe we should avoid the label prop for this control and add it explicitly here using another tag.